[2024-04-15T09:41:46.174+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:41:43.229749+00:00 [queued]>
[2024-04-15T09:41:46.186+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:41:43.229749+00:00 [queued]>
[2024-04-15T09:41:46.187+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-15T09:41:46.202+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data_from_text> on 2024-04-15 09:41:43.229749+00:00
[2024-04-15T09:41:46.209+0000] {standard_task_runner.py:60} INFO - Started process 1262 to run task
[2024-04-15T09:41:46.211+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'extract_data_from_text', 'manual__2024-04-15T09:41:43.229749+00:00', '--job-id', '1531', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmp8p69owuo']
[2024-04-15T09:41:46.213+0000] {standard_task_runner.py:88} INFO - Job 1531: Subtask extract_data_from_text
[2024-04-15T09:41:46.255+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:41:43.229749+00:00 [running]> on host 41da1b14e620
[2024-04-15T09:41:46.341+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='extract_data_from_text' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T09:41:43.229749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-15T09:41:43.229749+00:00'
[2024-04-15T09:41:46.343+0000] {python.py:202} INFO - Done. Returned value was: file does not exist
[2024-04-15T09:41:46.529+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=extract_data_from_text, execution_date=20240415T094143, start_date=20240415T094146, end_date=20240415T094146
[2024-04-15T09:41:46.586+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-15T09:41:46.610+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
